src/ripple/nodestore/impl/ManagerImp.cpp

```diff
@@ -21,8 +21,10 @@
 #include <ripple/nodestore/impl/ManagerImp.h>
 
 #include <boost/algorithm/string/predicate.hpp>
+#include <memory>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed